.github/workflows/Perf_Env_E2E_Test_CI.yml

```diff
@@ -84,7 +84,7 @@ jobs:
           QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
           QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
           IBM_API_KEY: ${{ secrets.IBM_API_KEY }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GIT_TOKEN }}
```

Comment:
https://github.com/kube-burner/kube-burner/blob/ead45c5903b03f8c04197d7a01a0a594fe75ba3e/.github/workflows/gorelease.yml#L27
check this, they did the same